webdriver-spec.html

```diff
@@ -1435,288 +1435,506 @@
+  implicitly trusted on <a data-lt=go>navigation</a> for the duration of the
+  session.
+  </tr>
 
-<aside class=example>
- <p>Capabilities may for example be used
-  to provide a serialised profile for a <a>remote end</a> to use
-  during startup of the user agent binary.
+  <tr>
+  <td><dfn>Page Load Strategy</dfn>
+  <td>"<code>pageLoadStrategy</code>"
+  <td>string
+  <td>Defines the <a>current session</a>’s <a>page load strategy</a>.
+  </tr>
 
- <p>A fictive example of this:
+  <tr>
+  <td><dfn>Proxy Configuration</dfn>
+  <td>"<code>proxy</code>"
+  <td>JSON Object
+  <td>Describes a session's proxy configuration.
+  </tr>
 
- <pre><code>capabilities := {}
-profile := serialise_profile("/path/to/profile")
-capabilities["profile"] = profile
-session := new_session(capabilities)</code></pre>
-</aside>
+  <tr>
+  <td><dfn>Session Timeouts Configuration</dfn>
+  <td>"<code>timeouts</code>"
+  <td>JSON Object
+  <td>Describes the timeouts imposed on certain session operation.
+  </tr>
+</tbody>
+</table>
 
-<p>When <dfn>processing capabilities</dfn> with argument <var>parameters</var>
- a <a>remote end</a> must run the following steps:
+<section>
+<h3>Proxy Configurations</h3>
+<p>The <a>Proxy Configuration</a> capability is a JSON Object nested within the
+primary <a>capabilities</a>. Implementations <em>MAY</em> define additional
+proxy configuration options, but they <em>MUST NOT</em> alter the
+semantics of those listed below.
 
-<ol>
- <li><p>Let <var>server capabilities</var> be a JSON Object
-  with the following entries:
+<table class=simple>
+<thead>
+<tr>
+<th>Key
+<th>Value Type
+<th>Description
+</tr>
+</thead>
+
+<tbody>
+<tr>
+<td>"<code>proxyType</code>"
+<td>string
+<td>Indicates the type of proxy configuration. This value <em>SHOULD</em> be one
+of the following: "<code>pac</code>", "<code>noproxy</code>",
+"<code>autodetect</code>", "<code>system</code>", or "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>proxyAutoconfigUrl</code>"
+<td>string
+<td>Defines the URL for a proxy auto-config file. [URL]
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>pac</code>".
+</tr>
+
+<tr>
+<td>"<code>ftpProxy</code>"
+<td>string
+<td>Defines the proxy <em>hostname</em> for FTP traffic. Alternatively, an
+  implementation <em>MAY</em> accept a hostname and port through this property.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>ftpProxyPort</code>"
+<td>number
+<td>Defines the proxy <em>port</em> for FTP traffic.
+  This property <em>SHOULD</em> only be set when the "<code>proxyType</code>" is
+  equal to "<code>manual</code>".
+</tr>
+
+<tr>
+<td>"<code>httpProxy</code>"
+<td>string
+<td>Defines the proxy <em>hostname</em> for HTTP traffic. Alternatively, an
+  implementation <em>MAY</em> accept a hostname and port through this property.
```

Comment:
I'd suggest raising an error, explaining the clash. Let the user make up their mind rather than guessing.